### ISSUE TITLE
WebDAV provider reports hundreds of connection attemps per second from SyncAny

### ISSUE THREAD

**Issue by BenoitJGirard** [syncany/syncany#472]
Got the following email from my WebDAV provider regarding SyncAny:

> We noticed that your account started opening hundreds of connections per second to our webdav server from XX.XX.XX.XX. This rate of connections is excessive and is causing performance issues on our server. Here is a sample of the requests we are receiving:

> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307230 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307233 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307230 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307233 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "DELETE /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307185662 HTTP/1.1" 404 366 "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307230 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "DELETE /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307185662 HTTP/1.1" 404 366 "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307231 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307438 HTTP/1.1" 201 343 "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "DELETE /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307185800 HTTP/1.1" 404 366 "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307230 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"
> XX.XX.XX.XX - someuser [29/Apr/2015:06:35:08 -0500] "PUT /someuser/syncany/org-synced/actions/action-down-vTvfgQYahknsDrKRsIjd-1430307307232 HTTP/1.1" 204 - "-" "Sardine/UNAVAILABLE"

> We have temporarily blocked access to XX.XX.XX.XX. Please adjust your application to use a reasonable number of concurrent connections and let me know when you're ready to try again.

**Comment by binwiederhier** [comment]
Oh boy. Apologies for the inconveniences.
Do you happen to have a detailed log from that client?

**Comment by BenoitJGirard** [comment]
No, I've deleted my setup since. I'll make sure to save the log if I encounter the problem again.

**Comment by pimotte** [comment]
@BenoitJGirard Can you tell me if your setup used to involve two pieces of syncing software (Syncany and something else?) This question is prompted by someone in IRC having a similar problem, and it reminded me of this issue.

**Comment by BenoitJGirard** [comment]
The WebDAV also hosts a KeePass2 password manager file, but which syncs only when manually triggered. 

In the SyncAny folders themselves, I do use applications with auto-save features such as Scrivener. 

--
Benoit.
